helpers/test/focus.test.js

```diff
@@ -358,4 +376,32 @@ describe('focus', () => {
 
 	});
 
+	describe('tryApplyFocus', () => {
+
+		it('returns true on single focusable element', async() => {
+			const elem = await fixture(focusableFixture);
+			expect(tryApplyFocus(elem.querySelector('button'))).to.be.true;
+		});
+
+		it('returns false on non-existent element', async() => {
```

Comment:
I think this the only test we can do to produce a false result. Since we keep checking the ancestors, we eventually bubble up to the focusable body in any non-null case.